utils/merge_and_push_translations.sh

```diff
@@ -54,89 +54,141 @@ fi
 
 echo "Working tree is clean, continuing"
 
-# Print all commands.
-set -x
+update_authors() {
+  # Update authors file
+  if python3 utils/update_authors.py; then
+    echo "Updated authors"
+  else
+    echo "Failed updating authors"
+    exit 1
+  fi
+  # Fix formatting for Lua
+  python3 utils/fix_formatting.py --lua --dir data/i18n
+  python3 utils/fix_formatting.py --lua --dir data/txts
+}
 
-# Update source catalogs
-python3 utils/buildcat.py
+update_appdata() {
```

Comment:
This would be fine for the 1st ("translation") commit.
But it does not catch the change on release, like in 5665009f6. And other potential manual changes to the template files. (see `d)` in [my comment](#issuecomment-1685073340))